### ISSUE TITLE
AttributeError: module 'torch.jit' has no attribute '_script_if_tracing'

### ISSUE THREAD

**Issue by gnthibault** [pytorch/pytorch#42227]
Dear all, for some reason, I recently started to have problems with torchvision, although it looks like I haven't changed anything recently.

Do you have any idea what happened ?
Thank you in advance for your help

```
    import segmentation_models_pytorch as smp
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/__init__.py", line 1, in <module>
    from .unet import Unet
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/unet/__init__.py", line 1, in <module>
    from .model import Unet
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/unet/model.py", line 3, in <module>
    from ..encoders import get_encoder
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/encoders/__init__.py", line 4, in <module>
    from .resnet import resnet_encoders
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/encoders/resnet.py", line 28, in <module>
    from torchvision.models.resnet import ResNet
  File "/usr/local/lib/python3.6/dist-packages/torchvision/__init__.py", line 5, in <module>
    from torchvision import models
  File "/usr/local/lib/python3.6/dist-packages/torchvision/models/__init__.py", line 12, in <module>
    from . import detection
  File "/usr/local/lib/python3.6/dist-packages/torchvision/models/detection/__init__.py", line 1, in <module>
    from .faster_rcnn import *
  File "/usr/local/lib/python3.6/dist-packages/torchvision/models/detection/faster_rcnn.py", line 7, in <module>
    from torchvision.ops import misc as misc_nn_ops
  File "/usr/local/lib/python3.6/dist-packages/torchvision/ops/__init__.py", line 1, in <module>
    from .boxes import nms, box_iou
  File "/usr/local/lib/python3.6/dist-packages/torchvision/ops/boxes.py", line 43, in <module>
    @torch.jit._script_if_tracing
AttributeError: module 'torch.jit' has no attribute '_script_if_tracing'
```

Which versions am I using:

> pip freeze | grep torch
> efficientnet-pytorch==0.6.3
> segmentation-models-pytorch==0.1.0
> torch==1.4.0
> torchvision==0.5.0

cc @suo @gmagogsfm

**Comment by wconstab** [comment]
This is a duplicate of #42078.  Closing issue

**Comment by pkliui** [comment]
I have had python version fixed to 3.6.13 in requirements. Removing the python version resolved the conflict. Here is the list my requirements:

dependencies:
  - ddt
  - furo
  - jupyterlab
  - matplotlib
  - myst-parser
  - numpy
  - opencv
  - pillow
  - pip
  - pytest
  - python#==3.6.13 - commenting out the version resolved the error
  - pytorch
  - scikit-image
  - scikit-learn
  - sphinx==4.0 # for furo to work
  - sphinx-togglebutton
  - torchvision
  - wandb
  - yaml
  - pip:
    - ray
